projects/1.19/assets/such-a-delight/suchadelight/lang/zh_cn.json

```
@@ -0,0 +1,234 @@
+  "block.suchadelight.mystic_fruit_cask_white": "箱装白色神秘果实",
+  "block.suchadelight.mystic_fruit_cask_red": "箱装红色神秘果实",
+  "block.suchadelight.mystic_fruit_cask_gold": "箱装金色神秘果实",
+
+  "block.suchadelight.cinnamon_log": "肉桂原木",
+  "block.suchadelight.cinnamon_wood": "肉桂木",
+  "block.suchadelight.stripped_cinnamon_log": "去皮肉桂原木",
+  "block.suchadelight.stripped_cinnamon_wood": "去皮肉桂木",
+  "block.suchadelight.cinnamon_leaves": "肉桂树叶",
+  "block.suchadelight.cinnamon_planks": "肉桂木板",
+  "block.suchadelight.cinnamon_door": "肉桂木门",
+  "block.suchadelight.cinnamon_trapdoor": "肉桂木活板门",
+  "block.suchadelight.cinnamon_ladder": "肉桂木梯",
+  "block.suchadelight.cinnamon_pressure_plate": "肉桂木压力板",
+  "block.suchadelight.cinnamon_button": "肉桂木按钮",
+  "block.suchadelight.cinnamon_sapling": "肉桂树苗",
+
+  "block.suchadelight.fried_chicken_dinner": "炸鸡晚餐",
+  "block.suchadelight.enchiladas": "辣酱玉米饼馅",
+  "block.suchadelight.pineapple_curry_rice": "菠萝咖喱饭",
+  "block.suchadelight.pizza_tower": "披萨塔",
+  "block.suchadelight.salmon_en_croute": "鲑鱼三明治",
+  "block.suchadelight.school_lunch_platter": "学校午餐拼盘",
+
+  "item.suchadelight.applesauce": "苹果酱",
+  "item.suchadelight.autumn_rolls": "秋天卷",
+  "item.suchadelight.baked_apples": "烤苹果",
+  "item.suchadelight.baked_beans": "烤豆子",
+  "item.suchadelight.beef_taco": "牛肉炸玉米饼",
+  "item.suchadelight.black_tea": "黑茶",
+  "item.suchadelight.blue_cheese_slice": "蓝纹芝士片",
+  "block.suchadelight.blue_cheese_wheel": "蓝纹奶酪圈",
+  "item.suchadelight.butterscotch_pie_slice": "奶油糖果馅饼片",
+  "block.suchadelight.butterscotch_pie": "奶油糖果馅饼",
+  "item.suchadelight.century_egg": "世纪彩蛋",
+  "item.suchadelight.cheese_balls": "芝士球",
+  "item.suchadelight.cheese_slice": "芝士片",
+  "block.suchadelight.cheese_wheel": "芝士圈",
+  "item.suchadelight.cheesecake_empanadas": "芝士蛋糕肉馅卷饼",
+  "item.suchadelight.chickpea_cakes": "鹰嘴豆蛋糕",
+  "item.suchadelight.chili": "辣椒",
+  "item.suchadelight.chocolate_bark": "巧克力树皮",
+  "item.suchadelight.chopped_cucumber": "切碎的黄瓜",
+  "item.suchadelight.choral_cheese_slice": "合唱奶酪片",
+  "block.suchadelight.choral_cheese_wheel": "合唱奶酪圈",
+  "item.suchadelight.cinnamon_roll": "肉桂捲",
+  "item.suchadelight.coffee_cake": "咖啡蛋糕",
+  "item.suchadelight.coffee": "咖啡",
+  "item.suchadelight.corn_chips": "玉米片",
+  "item.suchadelight.corn_chowder": "玉米杂烩",
+  "item.suchadelight.corn_pudding": "玉米布丁",
+  "item.suchadelight.cornbread": "玉米面包",
+  "item.suchadelight.corned_beef": "咸牛肉",
+  "item.suchadelight.crimson_cheese_slice": "绯红芝士片",
+  "block.suchadelight.crimson_cheese_wheel": "绯红芝士圈",
+  "item.suchadelight.crimson_tea": "绯红茶",
+  "item.suchadelight.egg_salad": "鸡蛋沙拉",
+  "item.suchadelight.eggplant_bolognese": "茄子肉酱",
+  "item.suchadelight.eggplant_burger": "茄子汉堡",
+  "item.suchadelight.eggplant_lasagna": "茄子千层面",
+  "item.suchadelight.eggplant_parmesan": "茄子帕尔马干酪",
+  "item.suchadelight.elotes": "Elotes",
+  "item.suchadelight.enchilada": "辣酱玉米饼馅",
+  "item.suchadelight.fajitas": "法吉塔斯",
```

```suggestion
  "item.suchadelight.fajitas": "墨西哥烤肉卷",
```
墨西哥烤肉来源维基百科，卷是因为游戏内确实是个卷，长得挺像老北京鸡肉卷（